bosch_locator_bridge/src/rosmsgs_datagram_converter.cpp

```diff
@@ -483,7 +483,7 @@ Poco::Buffer<char> RosMsgsDatagramConverter::convertLaserScan2DataGram(
   // angleInc
   writer << static_cast<float>(msg->angle_increment);
   // minRange
-  writer << static_cast<float>(msg->range_min);
+  writer << static_cast<float>(std::max(0., msg->range_min));
```

Comment:
```suggestion
  writer << static_cast<float>(std::max(0.0f, msg->range_min));
```
Use float zero to fix error of mismatched types